nginx/self-signed-cert.sls

```diff
@@ -0,0 +1,25 @@
+self_signed_cert:
+  module.run:
+    - name: tls.create_self_signed_cert
+    - CN: "journals.jcu.edu.au"
+    - ca.cert_base_path: "/etc"
+    - ca_name: "ssl"
+
+move cert:
+  file.rename:
+    - source: "/etc/pki/tls/certs/journals.jcu.edu.au.crt"
+    - name: "/etc/nginx/ssl/journals.jcu.edu.au.crt"
+    - makedirs: True
+  require:
```

Comment:
This should be a sub-item under the `file.rename` state, like:

```yaml
- require:
  - module: {{ host }} self-signed cert
```

Reply:
As above, this has been implemented.